app/code/Murergrej/Category/view/frontend/templates/category/fqa_content.phtml

```
@@ -25,6 +25,25 @@ if ($description = $block->getCurrentCategory()->getData('fqa_category')) :
     $dom = new \DOMDocument();
     @$dom->loadHTML('<?xml encoding="utf-8" ?>' . $descriptionHtml);
```

Consider returning an appropriate response for cases where no FAQs are available instead of silently failing.